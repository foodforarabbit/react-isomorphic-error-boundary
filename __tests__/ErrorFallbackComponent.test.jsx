import React from 'react';
import { shallow } from 'enzyme';
import FallbackErrorComponent from '../src/ErrorFallbackComponent';

describe('<ErrorFallbackComponent />', () => {
  let ErrorFallbackComponentTag = '';
  const props = {};

  test('renders correctly', () => {
    ErrorFallbackComponentTag = shallow(<FallbackErrorComponent {...props} />);
    expect(ErrorFallbackComponentTag).toMatchSnapshot();
  });
});
