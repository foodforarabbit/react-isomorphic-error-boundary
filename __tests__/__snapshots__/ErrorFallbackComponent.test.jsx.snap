// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorFallbackComponent /> renders correctly 1`] = `ShallowWrapper {}`;
